src/Tests/Microsoft.NET.Build.Containers.IntegrationTests/EndToEndTests.cs

```diff
@@ -26,13 +27,13 @@ public EndToEndTests(ITestOutputHelper testOutput)
 
     public static string NewImageName([CallerMemberName] string callerMemberName = "")
```

Comment:
nit
```suggestion
    public static string GetNewImageName([CallerMemberName] string callerMemberName = "")
```

Reply:
I'd rather not do the rename here to prevent additional churn if that's ok.